config/samples/flows_v1alpha1_flowcollector.yaml

```diff
@@ -30,6 +30,10 @@ spec:
     enableKubeProbes: true
     healthPort: 8080
     prometheusPort: 9090
+  kafka:
+    enable: false
+    address: "kafka-cluster-kafka-bootstrap"
+    topic: "flp-topic"
```

Comment:
as this CR will define the default values in OLM, I'd use a topic name that better describes what it is for. Maybe "netflows" ?

Reply:
Actually, `network-flows` would be better

Reply:
Done, thanks!